### ISSUE TITLE
Ruff barfs on the `match` statement

### ISSUE THREAD

**Issue by sfermigier** [charliermarsh/ruff#2686]
Starting with version 0.0.241, ruff rejects code containing `match` statements like:

```python
x = 0

match x:
  case 1: print("hello")
```

With error like:

```
error: Failed to parse toto.py: invalid syntax. Got unexpected token 'x' at line 3 column 7
```

(this passes with ruff 0.0.240).

**Comment by g-as** [comment]
It wasn't working before, it's just that since 0.0.241, you know about it ;)

#2505 was shipped in 0.0.241.

See the discussion in the PR.

**Comment by g-as** [comment]
Note that `ruff` exits 0 in both versions.

**Comment by g-as** [comment]
See #282.

**Comment by charliermarsh** [comment]
Yeah this is intended behavior. It was failing silently for you before.

**Comment by charliermarsh** [comment]
Fixed as of v0.0.250.
